issuance/issuance.go

```diff
@@ -69,6 +69,22 @@ type IssuerConfig struct {
 	Location IssuerLoc
 }
 
+// Certificate embeds an *x509.Certificate and represent the added semantics
```

Comment:
nit: I'd prefer that this be defined closer to the line 351-361 range, because the relationship is Issuer Has A Certificate, rather than up here where it is breaking up the various pieces of a ProfileConfig. But that's just me?